### PR TITLE
verhoeven2021replication

### PR DESCRIPTION
**Pull request by IvoOVerhoeven**
Sorry for the late commit, we've had plenty of issues with compilation on our side. Unfortunately, we never managed to get $make to work, and instead compiled in the provided overleaf project: https://www.overleaf.com/read/cbmrrhbwsmtd
I've edited the metadata.tex file according to what's present in the yaml file.

### CODE REVIEW

File: verhoeven2021replication/openreview/content.tex [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+
+%%%%%%%%%%%%%%%%%%%%%%at%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Experimental Setup and Code}
+\label{sec:experiments}
+\begin{figure}[t]
+\centering
+    \begin{subfigure}[t]{.23\linewidth}
+        \includegraphics[width=.9\textwidth]{openreview/pictures/Figure3/alpha_1.png}
```
Comment (koustuvsinha) [inline verhoeven2021replication/openreview/content.tex:86]: The compilation issues are due to the incorrect file path. When you compile using `make` from the `journal` folder, the latex compiler should expect the relative path of `../openreview/pictures/Figure3/alpha_1.png`. Please fix all the relative paths and then run `make` from journal folder.

**Review by koustuvsinha — COMMENTED** [review]
Please fix the paths of images to relative paths `../openreview`

**Comment by IvoOVerhoeven** [comment]
I changed all the image paths, openreview compiles locally, make still not working.

File: verhoeven2021replication/openreview/content.tex [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+        \caption{Sweep $\alpha_1$}
+        % \label{fig:alpha_1}
+    \end{subfigure}
+    \begin{subfigure}[t]{.23\linewidth}
+        \includegraphics[width=.9\textwidth]{openreview/pictures/Figure3/beta_1.png}
```
Comment (koustuvsinha) [inline verhoeven2021replication/openreview/content.tex:91]: Not all paths are changed yet, which results in compilation failure

File: verhoeven2021replication/openreview/content.tex (hunk 1 above)
Comment (koustuvsinha) [reply verhoeven2021replication/openreview/content.tex:86]: Still not changed it seems

**Comment by IvoOVerhoeven** [comment]
The correct openreview/content file should have been pushed now

**Comment by koustuvsinha** [comment]
Compilation fails: 

```
Latexmk: Missing input file: '../openreview/ablation study 034/InformationFlow.png' from line
  'LaTeX Warning: File `../openreview/ablation study 034/InformationFlow.png' not found on input line 231.'
```

Most probably the folder names are an issue here. Can you change the folder names to a "_" delimited name instead of space? Eg, convert `ablation study 034` to `ablation_study_034`

File: verhoeven2021replication/journal/metadata.yaml [hunk 3]
```diff
@@ -5,59 +5,51 @@
...
-    affiliations: 1,2
-    
-  - name: Nicolas P. Rougier
-    orcid: 0000-0002-6972-589X
-    email: Nicolas.Rougier@inria.fr
-    affiliations: 3,4,5,*      # * is for contact author
+  - name: Ivo Verhoeven
+    orcid: 0000-0002-5163-210X
+    email: ivo.verhoeven@student.uva.nl
+    affiliations: 1
```
Comment (koustuvsinha) [inline verhoeven2021replication/journal/metadata.yaml:16]: Please add a '*' (separated by comma) in the affiliations if you are the primary contact. 

```
affiliations: 1,*
```

**Comment by IvoOVerhoeven** [comment]
Directory names should all have underscores now

**Comment by koustuvsinha** [comment]
@IvoOVerhoeven Thanks! please also add the contact author information ('*') for the build to compile.

**Comment by IvoOVerhoeven** [comment]
I've made some small edits to the metadata file, but the contact author * should already be present in the metadata file.

File: verhoeven2021replication/journal/metadata.yaml [hunk 4]
```diff
@@ -5,59 +5,51 @@
...
-    affiliations: 1,2
-    
-  - name: Nicolas P. Rougier
-    orcid: 0000-0002-6972-589X
-    email: Nicolas.Rougier@inria.fr
-    affiliations: 3,4,5,*      # * is for contact author
+  - name: Ivo Verhoeven
+    orcid: 0000-0002-5163-210X
+    email: ivo.verhoeven@student.uva.nl
+    affiliations: 1, *
```
Comment (koustuvsinha) [inline verhoeven2021replication/journal/metadata.yaml:16]: remove the space between 1, and *

File: verhoeven2021replication/journal/metadata.yaml [hunk 5]
```diff
@@ -5,59 +5,51 @@
...
+authors:
```
Comment (koustuvsinha) [inline verhoeven2021replication/journal/metadata.yaml:17]: This yaml is malformed. `authors` is a list, which contains objects. You have created multiple `authors` lists.. please remove this line (as `authors` list is defined in L12)

File: verhoeven2021replication/journal/metadata.yaml [hunk 6]
```diff
@@ -5,59 +5,51 @@
...
+  - name:  Xinyi Chen
+    orcid: 0000-0001-8757-344X
+    email: xinyi.chen@student.uva.nl
+    affiliations: 1
+authors:
```
Comment (koustuvsinha) [inline verhoeven2021replication/journal/metadata.yaml:22]: please remove this line (as `authors` list is defined in L12)

File: verhoeven2021replication/journal/metadata.yaml [hunk 7]
```diff
@@ -5,59 +5,51 @@
...
+  - name: Qingzhi Hu
+    orcid: 0000-0001-7419-2031
+    email: qingzhi.hu@student.uva.nl
+    affiliations: 1
+authors:
```
Comment (koustuvsinha) [inline verhoeven2021replication/journal/metadata.yaml:27]: please remove this line (as `authors` list is defined in L12)

**Review by koustuvsinha — CHANGES_REQUESTED** [review]

**Comment by IvoOVerhoeven** [comment]
Edits have been made

**Comment by koustuvsinha** [comment]
Looks good to me! Compiles correctly now.
@sashavor can you review the contents of the paper?

**Review by koustuvsinha — APPROVED** [review]

**Comment by koustuvsinha** [comment]
@IvoOVerhoeven while going over the contents of the paper I realize you have not added the Reproducibility Summary in the first page. Please add it as soon as possible.

**Comment by koustuvsinha** [comment]
@IvoOVerhoeven please act on this asap!

**Comment by IvoOVerhoeven** [comment]
Sorry about that, we initially had it as an abstract. The summary should be on the first 2 pages now without large page gaps in the report.

File: verhoeven2021replication/openreview/content.tex [hunk 8]
```diff
@@ -81,29 +120,29 @@ \subsection{Implementation}
...
-    \begin{subfigure}[t]{.23\linewidth}
-        \includegraphics[width=.9\textwidth]{../openreview/pictures/Figure3/beta_3.png}
-        \caption{Sweep $\beta_3$}
-        % \label{fig:beta_2}
-    \end{subfigure}
-    \caption{Visualizations of learned latent factors for MNIST classifier trained on classes `3' and `8'. The colour denotes the classifier's decision, with each adhering to a specific class. The axis  gives the additive value to each latent dimension. Complete results are in Figure \ref{fig:complte_:mnist_results 38} for MNIST 3-8, Figure \ref{fig:mnist_results 149 complete} for MNIST 1-4-9, and Figure \ref{fig:fmnist_results complete} for FMNIST 0-3-4.}
-    \label{fig:mnist_results 38}
+	\centering
+	\begin{subfigure}[t]{.23\linewidth}
+		\includegraphics[width=.9\textwidth]{openreview/pictures/Figure3/alpha_1.png}
```
Comment (koustuvsinha) [inline verhoeven2021replication/openreview/content.tex:125]: Why did you re-introduce the bug? I thought we discussed this.

File: verhoeven2021replication/openreview/content.tex [hunk 9]
```diff
@@ -226,31 +264,31 @@ \subsection{Results reproducing original paper}
...
-    \caption{Ablation study for training FMNIST dataset: (a) each latent factor affecting the classifier performance measured by information flow. (b) comparison of classifier accuracies when data aspects regulated by the individual latent factors are eliminated. Figure 5(c-d) visualizes the aspects learned by $\alpha_1$ and $\beta_1$. The figures here are generated using the FMNIST data-set.}
-    \label{fig:ablation information flow}
+	\centering
+	\begin{subfigure}[t]{.45\linewidth}
+		\includegraphics[width=.9\textwidth]{openreview/ablation study 034/InformationFlow.png}
+		\caption{Information flow.}
+		% \label{fig:5a}
+	\end{subfigure}
+	\begin{subfigure}[t]{.45\linewidth}
+		\includegraphics[width=.9\textwidth]{openreview/ablation study 034/AblationAccuracy.png}
```
Comment (koustuvsinha) [inline verhoeven2021replication/openreview/content.tex:274]: Why did you re-introduce the folder naming bug? I thought we discussed this.

**Review by koustuvsinha — CHANGES_REQUESTED** [review]
In your last commit you re-introduced two errors which we fixed before:
1. relative paths (../openreview)
2. folder names.

Please check again.

**Comment by IvoOVerhoeven** [comment]
I've rolled it back to the last accepted version except for the addition of the summary.

**Review by koustuvsinha — APPROVED** [review]
Thank you, compilation works now and paper has all necessary components.

**Pull request merged**